### PR TITLE
KIALI-2384 Code cleaning & consistent naming in "config" dir

### PR DESCRIPTION
**Pull request by jotak**
Fix some inconsistencies, we currently have exported consts:
- PascalCase (ex: KialiLogo)
- UPCASE (ex: ICONS)
- camelCase (ex: config)

So, moving all to camelCase in this "config" directory. Note, to avoid very large PRs across all the code, this change is focused only on config.

Also fixing:
- IstioLogo isn't used anymore, so removing its definition
- deepFreeze was called for each invocation of the resource (config and/or icons) whereas this is something static. So calling only once.

Backend PR required: https://github.com/kiali/kiali/pull/839

### CODE REVIEW

File: src/config/Config.ts [hunk 1]
```diff
@@ -126,20 +126,20 @@ const conf = {
   }
 };
 
-export const config = () => {
-  return deepFreeze(conf) as typeof conf;
-};
+export const config = deepFreeze(conf) as typeof conf;
```
Comment (jotak) [inline src/config/Config.ts]: Note about that: I don't know if there was a good reason to have this lazy `deepFreeze`, repeated for each access. Testing locally, it doesn't seem to break anything to set it once for all. Anyone has an idea about the reason? (@israel-hdez , @mtho11  maybe? )

Comment (israel-hdez) [reply src/config/Config.ts]: Not sure. `conf` is also a constant, so it shouldn't mutate. For me, it is better with your change.

**Comment by jotak** [comment]
@jshaughn they export respectively `serverConfig` (the function, not the interface which is defined elsewhere) and `config`, so in both cases it would be camelCase, seems to me.

**Comment by jshaughn** [comment]
@jotak Ah, OK.  So it's named after the export.  So it makes sense.  As long as the export is properly cased, I'll assume it is :)

**Comment by jotak** [comment]
@jshaughn yes... I hope this rule of "named after the export" will be fine in the long run. It was already "kind of" followed by some of us I think, which is not too surprising as for java developers we're used to having files named after their public class.

**Pull request merged**